docs(airgap): deploy/upgrade processes (#7550)
vatesfr/xen-orchestra: docs/installation.md

# Installation

:::tip

> If you want to deploy an official XO Appliance on an airgapped infrastructure, refer to the [dedicated documentation](airgap.md).

Reviewer: The use of "official" feels out of place, especially since it isn't used elsewhere beyond the new airgap.md. My suggestion would be to change "an official" to "the"

Author: I will go with *an XOA* to better match the rest of the docs then.

```suggestion
If you want to deploy an XOA in an airgapped infrastructure, refer to the [dedicated documentation](airgap.md).
```

Reviewer 2: I would change "on an airgapped infrastructure" to "in an airgapped infrastructure", this is how we typically say this in english
